gateway/assets/src/error.html

```diff
@@ -0,0 +1,72 @@
+  <title>{{ .StatusCode }} {{ .StatusText }}</title>
+  <link rel="stylesheet">
+</head>
+<body>
+  <header></header>
+  <div id="content">
+    <div id="content-header">
+      <strong>{{ .StatusCode }} {{ .StatusText }}</strong>
+    </div>
+
+    <div class="table-responsive">
+      <table>
+        <tr>
+          <td>
+            {{ if eq .StatusCode 400 }}
+              <p>Your request is invalid. Please check the error below for more information.</p>
+            {{ else if eq .StatusCode 404 }}
+              <p>The content path you requested cannot be found. There's likely an invalid or missing DAG node.</p>
+            {{ else if eq .StatusCode 410 }}
+              <p>The content path you tride to request is correct, but this gateway refuses to return the requested data.</p>
+            {{ else if eq .StatusCode 412 }}
+              <p>This gateway is unable to return the requested data under the conditions sent by the client.</p>
+            {{ else if eq .StatusCode 429 }}
+              <p>You have made too many requests. Please wait some time and try again.</p>
+            {{ else if eq .StatusCode 451 }}
+              <p>This gateway is not allowed to return the requested data due to legal reasons.</p>
+            {{ else if eq .StatusCode 500 }}
+              <p>This gateway was unable to return the requested data due to an internal error. Please check the error below for more information.</p>
+            {{ else if eq .StatusCode 504 }}
+              <p>This gateway was unable to fullfil your request within internally defined time limits.</p>
```

Comment:
These two will be constantly shown on Rhea / bifrost-gateway, so worth giving them useful messages, but at the same time we should not make then about Saturn:

- 502 is when IPNI (cid.contact) fails to find providers
- 504 is when Saturn L1  failed to retrieve data under the timeout 


Perhaps:
```suggestion
            {{ else if eq .StatusCode 502 }}
              <p>The gateway backed was unable to fullfil your request due to an error.</p>
            {{ else if eq .StatusCode 504 }}
              <p>The gateway backend was unable to fullfil your request due to a timeout.</p>
```